pkg/services/shorturls/short_url_service.go

```diff
@@ -0,0 +1,61 @@
+package shorturls
+
+import (
+	"github.com/grafana/grafana/pkg/bus"
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/util"
+)
+
+type ShortURLService interface {
+	GetFullURLByUID(uid string) (string, error)
+	CreateShortURL(path string) (string, error)
+}
+
+type shortURLServiceImpl struct {
+	user *models.SignedInUser
+}
+
+var NewShortURLService = func(user *models.SignedInUser) ShortURLService {
+	return &shortURLServiceImpl{
+		user: user,
+	}
+}
+
+func (dr *shortURLServiceImpl) buildCreateShortURLCommand(path string) (*models.CreateShortURLCommand, error) {
+	cmd := &models.CreateShortURLCommand{
+		OrgID:     dr.user.OrgId,
+		UID:       util.GenerateShortUID(),
+		Path:      path,
+		CreatedBy: dr.user.UserId,
+	}
+
+	return cmd, nil
+}
```

Comment:
Can you just inline this method please? It's super tiny and just makes the code more difficult to follow.